verticapy/tests_new/core/vdataframe/test_encoding.py

```diff
@@ -15,17 +15,43 @@
 permissions and limitations under the License.
 """
 from itertools import chain
-import pandas as pd
+
 import numpy as np
-from sklearn.preprocessing import LabelEncoder
+import pandas as pd
 import pytest
+from sklearn.preprocessing import LabelEncoder
 
 
```

Comment:
It maybe better to follow the same syntax as the original file:
There should be two classes;

TestVDFEncode

TestVDCEncode

Also you are missing one_hot_encode for vDF

Reply:
Done. divided into vdataframe and vColumn.
one_hot_encode is already there.